ts/screens/services/ServicesHomeScreen.tsx

```diff
@@ -378,32 +392,80 @@ class ServicesHomeScreen extends React.Component<Props, State> {
 const mapStateToProps = (state: GlobalState) => {
   const { services } = state.entities;
 
+  const potUserMetadata = userMetadataSelector(state);
+  // TODO: disable selection of areas of interest if the user metadata are not noaded
+  // (it causes the new selection is not loaded) https://www.pivotaltracker.com/story/show/168312476
```

Comment:
```suggestion
  // (it causes the new selection if it is not loaded) https://www.pivotaltracker.com/story/show/168312476
```
I'm not sure you want to say what I proposed

Reply:
I meant the selection made by the user is saved anywhere